library/rakudo-star

```diff
@@ -1,5 +1,7 @@
-# maintainer: Rob Hoelz <robAThoelz.ro> (@hoelzro)
+Maintainers: Rob Hoelz <robAThoelz.ro> (@hoelzro)
 
-2018.04:  git://github.com/perl6/docker@aff1beeb74853acb6d04c7335ef56c56f0270407
+GitRepo: https://github.com/perl6/docker
```

Comment:
There's a bit too much whitespace here.  To give some background, this newer format is essentially a series of what we call "entries", each of which is separated by a single blank line.  The first "entry" is the global entry, and provides the `Maintainers` and the defaults for any entry which doesn't specify values otherwise.  Each non-global entry is required to have at least `Tags:` (although if there's more than one non-global entry, specifying at least `Directory` or `GitFetch`/`GitCommit` is probably necessary).

Since you've only got the one version supported at a time, you could do any of the following:

```
Maintainers: Rob Hoelz <robAThoelz.ro> (@hoelzro)

Tags: latest, 2018.04
GitRepo: https://github.com/perl6/docker.git
Architectures: amd64, arm64v8
GitCommit: aff1beeb74853acb6d04c7335ef56c56f0270407
```

```
Maintainers: Rob Hoelz <robAThoelz.ro> (@hoelzro)
GitRepo: https://github.com/perl6/docker.git
GitCommit: aff1beeb74853acb6d04c7335ef56c56f0270407
Architectures: amd64, arm64v8

Tags: latest, 2018.04
```

```
Maintainers: Rob Hoelz <robAThoelz.ro> (@hoelzro)
GitRepo: https://github.com/perl6/docker.git
GitCommit: aff1beeb74853acb6d04c7335ef56c56f0270407

Tags: latest, 2018.04
Architectures: amd64, arm64v8
```

etc. etc.

Which you go with probably depends a little bit on whether you plan to support multiple versions of Rakudo Star concurrently in the future, but it's also pretty trivial to change so doesn't make much difference. :smile:

(Hopefully this helps explain the format better. :+1:)

Reply:
@tianon Yes, it does! Thanks for having a look; I'll fix this up and update the PR later today!

Reply:
@tianon Ok, I've submitted a change incorporating your suggestions!